### PR TITLE
[PD-2690] Editing the mobile solution for the logo alignment

### PR DESCRIPTION
**Pull request by mwarnsleyDE**
Editing the logo alignment in mobile and desktop view.

To Test:
In the admin panel, switch to any .jobs domain. Make sure you are on V2 templates and also that you are using the home_page_listing_bootstrap3.html template. Also it should be set to HTML5. Specifically domains with longer names should be used and check the logo and text in mobile and desktop view. Should be somewhat centered for most of the time. Essentially it is a quick fix for the problem as we take time to explore possible other options to handle longer and shorter names together.

### CODE REVIEW

File: static/seo-base-scripts.js [hunk 1]
```diff
@@ -245,13 +245,13 @@ JAVASCRIPT FOR THE NEW REBRANDING HOMEPAGE FUNCTIONALITY
...
-  var siteName = $('.site-name').text();
-
-  if(siteName.length > 10){
-    var currentPosLeft = 0;
-    var newLength = siteName.length - 10;
-    var posLeft = newLength * 4;
-    $('.site-name').css('left', currentPosLeft - posLeft + '%');
-  }
-})();
+// (function(){
```
Comment (tdphillips) [inline static/seo-base-scripts.js]: Should this be commented out?

Comment (mwarnsleyDE) [inline static/seo-base-scripts.js]: Yes, I commented it out to make sure that my other solution was okay first and then I plan on going back in and deleting it.

Comment (koonsies) [inline static/seo-base-scripts.js]: We typically don't commit commented code, as it has a way to getting forgotten about over time. What is a bit nicer is to simply use git to store the history.. Make a commit that has the code you may want in the future, then follow it with a commit that removes that code. Therefore, if you need it, it's in source control.. but not something you have to clean up later.

File: static/seo-base-scripts.js [hunk 2]
```diff
@@ -244,14 +244,3 @@ JAVASCRIPT FOR THE NEW REBRANDING HOMEPAGE FUNCTIONALITY
...
-(function(){
-  var siteName = $('.site-name').text();
-
-  if(siteName.length > 10){
-    var currentPosLeft = 0;
-    var newLength = siteName.length - 10;
-    var posLeft = newLength * 4;
-    $('.site-name').css('left', currentPosLeft - posLeft + '%');
-  }
-})();
```
Comment (koonsies) [inline static/seo-base-scripts.js]: 👍 much better.

**Comment by mwarnsleyDE** [comment]
@tdphillips @koonsies I've taken out the commented code from the Javascript file.

**Comment by mwarnsleyDE** [comment]
lol

**Comment by mwarnsleyDE** [comment]
However, I'm still not too sure as to why my test are failing because of the Python test?

**Comment by mwarnsleyDE** [comment]
@tdphillips @koonsies Any ideas why the python test keep failing on this one?

**Pull request merged**